commit da80cb206af751392e34bf6de36eef4818d4cfd7
Author:     Sean Yang <seany314@gmail.com>
AuthorDate: Mon Feb 7 15:30:10 2022 -0800
Commit:     GitHub <noreply@github.com>
CommitDate: Mon Feb 7 15:30:10 2022 -0800

    Standardize error messages (#178)
    
    * Standardize error messages
    
    * change asserts to TypeError, remove some unnecessary asserts
---
 nvflare/private/fed/client/client_executor.py | 4 +++-
 nvflare/private/fed/server/server_engine.py   | 6 ++++--
 2 files changed, 7 insertions(+), 3 deletions(-)

diff --git a/nvflare/private/fed/client/client_executor.py b/nvflare/private/fed/client/client_executor.py
index 8360c8fd..ac5561cb 100644
--- a/nvflare/private/fed/client/client_executor.py
+++ b/nvflare/private/fed/client/client_executor.py
@@ -25,6 +25,7 @@ from multiprocessing.connection import Client
 from nvflare.apis.fl_constant import AdminCommandNames, ReturnCode
 from nvflare.apis.shareable import Shareable, make_reply
 from nvflare.fuel.utils.pipe.file_pipe import FilePipe
+
 from .client_status import ClientStatus, get_status_message
 
 
@@ -185,7 +186,8 @@ class ProcessExecutor(ClientExecutor):
         command = (
             f"{sys.executable} -m nvflare.private.fed.app.client.worker_process -m "
             + args.workspace
-            + " -w " + self.startup
+            + " -w "
+            + self.startup
             + " -s fed_client.json "
             " --set" + command_options + " print_conf=True"
         )
diff --git a/nvflare/private/fed/server/server_engine.py b/nvflare/private/fed/server/server_engine.py
index 5cdb3de3..3c5d0837 100644
--- a/nvflare/private/fed/server/server_engine.py
+++ b/nvflare/private/fed/server/server_engine.py
@@ -68,7 +68,9 @@ class ServerEngine(ServerEngineInternalSpec):
 
         self.engine_info = EngineInfo()
 
-        assert workers >= 1, "workers must >= 1"
+        if not workers >= 1:
+            raise ValueError("workers must >= 1 but got {}".format(workers))
+
         self.executor = ThreadPoolExecutor(max_workers=workers)
         self.lock = Lock()
         self.logger = logging.getLogger(self.__class__.__name__)
@@ -287,7 +289,7 @@ class ServerEngine(ServerEngineInternalSpec):
 
     def set_configurator(self, conf: ServerJsonConfigurator):
         if not isinstance(conf, ServerJsonConfigurator):
-            raise TypeError("Must be a type of ServerJsonConfigurator.")
+            raise TypeError("conf must be ServerJsonConfigurator but got {}".format(type(conf)))
         self.conf = conf
 
     def build_component(self, config_dict):
